.vscode/settings.json

```diff
@@ -5,23 +5,27 @@
         "dev-env/lib/*": true,
         "bazel-*/*": true,
         ".bazel-cache/*": true,
-        "**/node_modules/*": true,
+        "**/node_modules/*": true
```

Comment:
Can you confirm this file is even being read? Dangling commas are not valid JSON AFAIK.

Reply:
`.vscode/settings.json` is definitely being read, I could see the changes in VSCode as I changed it.

VSCode must have some in-built support for dangling commas...